### PR TITLE
Ingestion: AACL-IJCNLP 2023

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

**Comment by mjpost** [comment]
Is this waiting for something? Just curious why it's still a draft.

**Comment by anthology-assist** [comment]
@mjpost #2791 has more volumes.

**Comment by mjpost** [comment]
(@anthology-assist Please see all the comments above and mark as resolved when addressed)

**Comment by anthology-assist** [comment]
@mjpost I think this is ready to be merged (although we are still waiting on 1 workshop material). I'm not seeing any comments other than it being a draft.

**Comment by FremyCompany** [comment]
It would be good to merge this soon :)

That said, something went wrong with the FinNLP workshop.
https://github.com/acl-org/acl-anthology/blob/00393be3251aa3f6365e237bf4c18c90b680d600/data/xml/2023.finnlp.xml

(My own workshop is fine, but a colleague of mine published in that workshop)

File: data/yaml/venues/nlint.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+acronym: nlint
+name: The Second Workshop on Natural Language Interfaces
```
Comment (mjpost) [inline data/yaml/venues/nlint.yaml:2]: Strike "The Second"

File: data/yaml/venues/sealp.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+acronym: sealp
+name: The First Workshop in South East Asian Language Processing
```
Comment (mjpost) [inline data/yaml/venues/sealp.yaml:2]: Strike "The First"

File: data/xml/2023.ijcnlp.xml [hunk 3]
```diff
@@ -0,0 +1,1269 @@
...
+      <editor><first>Baotian</first><last>Hu</last></editor>
+      <editor><first>Wei</first><last>Lu</last></editor>
+      <editor><first>Derry</first><last>Wijaya</last></editor>
+      <editor><first>Ayu</first><last>Purwarianti</last></editor>
+      <editor><first>Adila Alfa</first><last>Krisnadhi</last></editor>
+      <publisher>Association for Computational Linguistics</publisher>
+      <address>Nusa Dua, Bali</address>
+      <month>November</month>
+      <year>2023</year>
+      <venue>ijcnlp</venue>
```
[the remark this answers is not in the document]
Comment (mjpost) [reply data/xml/2023.ijcnlp.xml:17]: i.e., add `<venue>aacl</venue>`

**Comment by anthology-assist** [comment]
> It would be good to merge this soon :)
> 
> That said, something went wrong with the FinNLP workshop. https://github.com/acl-org/acl-anthology/blob/00393be3251aa3f6365e237bf4c18c90b680d600/data/xml/2023.finnlp.xml
> 
> (My own workshop is fine, but a colleague of mine published in that workshop)

@FremyCompany Could you elaborate what's wrong?

**Comment by anthology-assist** [comment]
> @anthology-assist can you please add all workshops to the `<event>` block for both IJCNLP and AACL?

Done

**Review by speedcell4 — APPROVED** [review]

**Review by mjpost — APPROVED** [review]

**Pull request merged**